packages/rpc-methods/src/permitted/invokeKeyring.ts

```diff
@@ -48,6 +49,8 @@ export type InvokeKeyringHooks = {
   }: SnapRpcHookArgs & { snapId: SnapId }) => Promise<unknown>;
 
   getSnap: (snapId: SnapId) => Snap | undefined;
+
+  getAllowedKeyringMethodsForOrigin: (origin: string) => string[];
```

Comment:
I thought we wanted to drop the `ForOrigin` suffix?

Reply:
Yes, indeed, I forgot to remove it 😅